.github/workflows/diff_report.yml

```diff
@@ -180,8 +185,11 @@ jobs:
         DIFF="./contribution/checkstyle-tester/reports/diff"
         LINK="https://${{env.AWS_BUCKET_NAME}}.s3.${{env.AWS_REGION}}.amazonaws.com"
         aws s3 cp $DIFF s3://${{env.AWS_BUCKET_NAME}}/$FOLDER/reports/diff/ --recursive
-        echo $LINK/$FOLDER/reports/diff/index.html > message
-        
+        if [ ! -z "${{needs.parse_body.outputs.report_label}}" ]; then
```

Comment:
Why `! -z` instead of `-n`?

Reply:
I didn't know about this. Thanks, changed. I also refactored other places where `! -z` was used

Reply:
Resolved.